src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1133,7 +1133,7 @@ limitations under the License.
                 <requireCommand>RemoveAllGroups</requireCommand>
                 <requireCommand>AddGroupIfIdentifying</requireCommand>
             </include>
-            <include cluster="Scenes" client="true" server="true" clientLocked="true" serverLocked="true">
+            <include cluster="Scenes" client="true" server="false" clientLocked="true" serverLocked="true">
                 <requireAttribute>SCENE_COUNT</requireAttribute>
```

Comment:
Should we have attributes in client clusters ? @dhrishi

Reply:
@tecimovic @bzbarsky-apple Do you know why attributes are present under client clusters in the XML? Should they be removed?

Reply:
Attributes in client clusters don't really make sense.  Please do a followup to remove that @jadhavrohit924 ?